contracts/FIFOMarket.sol

```diff
@@ -39,6 +47,8 @@ contract FIFOMarket is
   uint256 private _noriFee;
   uint256 public priorityRestrictedThreshold;
   uint256 public totalSupply;
+  mapping(uint256 => mapping(address => Fee)) feeAddressToFeeIndex;
```

Comment:
I guess "fee" makes sense to use here but I was confused at first trying to figure out what's going on here because it's not really a "fee" that we're giving suppliers, it's their actual payment...  I think of fee as like a percentage whereas payment would be the actual value due someone.  Not sure I have a better idea though... and "fee" is nice and short.

Can we rename this mapping `orderToAddressToFeeStruct` or something?  it's not mapping to an index but rather to an entire `Fee`

Reply:
oops yeah forgot to rename this after playing around with different data structures. `orderToAddressToFeeStruct` makes sense to me